### PR TITLE
Update expenses policy

### PR DESCRIPTION
**Pull request by rub1e**
Takes outdated and incomprehensible expenses policy out of Google Docs and into `finance.md` along with other similar info

Clarifies requirements of VAT receipt for all expenses 

Explains expenses submission process for PAYE dwylers vs freelancers

### CODE REVIEW

**Comment by nelsonic** [comment]
@rub1e please give a short summary of what has changed in the PR desc. 😉 
thanks. ✨

File: finances.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ able to see here what the amount due is for the latest payrun
   + For reference, the payment should equal the total of the following from the
    relevant payrun in Xero: Income tax, Student Loan deductions, Employee National
    Insurance Contribution (NIC), Employer NIC.  
-  + **NOTE:** We have an [employment allowance of £3,000 _annually_](https://www.gov.uk/claim-employment-allowance). This means that
+  + **NOTE:** We have an [employment allowance of £3,000_annually_](https://www.gov.uk/claim-employment-allowance). This means that
```
Comment (iteles) [inline finances.md]: @rub1e This reminds me, can we please just check wether Xero automatically deducts this for us all at once at the beginning of the year or whether it spreads it out?

Comment (rub1e) [reply finances.md]: It's split over the first couple of months - obviously depends on how much we're paying in NICs (i.e. how many employees, how much they're paid), but it gets used up pretty quickly

The bigger question is: why did I amend that line, in an unrelated commit, only to INTRODUCE a typo??? <picard_facepalm.jpg>

Will do a PR

Comment (rub1e) [reply finances.md]: OH! It's because the link was underlined which made the underscore of the italics look like a blank space

**Pull request merged**